firmware/tunerstudio/tunerstudio.template.ini

```diff
@@ -1702,18 +1702,18 @@ menuDialog = main
 		subMenu = stagedInjectionTbl,		"Staged injection % table", 0, {isInjectionEnabled && enableStagedInjection}
 
 		groupMenu = "Cylinder fuel trims"
-		groupChildMenu = fuelTrimTbl1,		"Fuel trim cyl 1"
-		groupChildMenu = fuelTrimTbl2,		"Fuel trim cyl 2"
-		groupChildMenu = fuelTrimTbl3,		"Fuel trim cyl 3"
-		groupChildMenu = fuelTrimTbl4,		"Fuel trim cyl 4"
-		groupChildMenu = fuelTrimTbl5,		"Fuel trim cyl 5"
-		groupChildMenu = fuelTrimTbl6,		"Fuel trim cyl 6"
-		groupChildMenu = fuelTrimTbl7,		"Fuel trim cyl 7"
-		groupChildMenu = fuelTrimTbl8,		"Fuel trim cyl 8"
-		groupChildMenu = fuelTrimTbl9,		"Fuel trim cyl 9"
-		groupChildMenu = fuelTrimTbl10,		"Fuel trim cyl 10"
-		groupChildMenu = fuelTrimTbl11,		"Fuel trim cyl 11"
-		groupChildMenu = fuelTrimTbl12,		"Fuel trim cyl 12"
+		groupChildMenu = fuelTrimTbl1,		"Fuel trim cyl 1"  { isInjectionEnabled == 1 && cylindersCount >= 1 }
```

Comment:
@alrijleh any chance you are missing commas?

Reply:
Hmm I see what you mean.  I'm surprised since the generated ini works as expected on my end.  I can add those in

Reply:
> Hmm I see what you mean. I'm surprised since the generated ini works as expected on my end. I can add those in

Maybe these commas are optional on purpose, maybe a happy TS bug

Reply:
the TS ini parser is very....interesting